Sources/StreamChat/StateLayer/MemberList.swift

```diff
@@ -40,8 +38,6 @@ public final class MemberList {
     
     /// Loads channel members for the specified pagination parameters and updates ``MemberListState/members``.
     ///
-    /// - Important: If pagination offset is 0 and cursor is nil, then loaded members are reset.
```

Comment:
Removed it because this is not implemented. Same with `ReactionList`. We can add it back when we have added support for this.